QEfficient/transformers/modeling_utils.py

```diff
@@ -114,6 +123,12 @@
     GPT2Block: QEffGPT2Block,
     GPT2Attention: QEffGPT2Attention,
     GPT2LMHeadModel: QEffGPT2LMHeadModel,
+    # Dbrx model layers
```

Comment:
Please add in `QEfficient/transformers/pytorch_transforms.py::KVCacheTransform` too.
We will be deprecating this after 1.18 release.